homeassistant/components/smart_meter_texas/config_flow.py

```diff
@@ -0,0 +1,79 @@
+"""Config flow for Smart Meter Texas integration."""
+import asyncio
+import logging
+
+from aiohttp import ClientError
+from smart_meter_texas.async_api import Auth, SMTError
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.helpers import aiohttp_client
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required("meter"): str,
```

Comment:
Can we call the https://www.smartmetertexas.com/api/meter api and get all the meters on startup instead of requiring it here?